src/ess/amor/load.py

```diff
@@ -142,21 +142,13 @@ def load_amor_ch_frequency(ch: RawChopper[RunType]) -> ChopperFrequency[RunType]
 def load_amor_sample_rotation(fp: Filename[RunType]) -> SampleRotation[RunType]:
     (mu,) = load_nx(fp, "NXentry/NXinstrument/master_parameters/mu")
     # For some reason this is length 1 and not scalar sometimes
-    if mu["value"].coords["average_value"].dims != ():
-        mu = mu["value"].coords["average_value"].values[0]
-    else:
-        mu = mu["value"].coords["average_value"].value
-    return sc.scalar(mu, unit="deg")
+    return sc.scalar(mu['value'].data['dim_1', 0]['time', 0].value, unit='deg')
```

Comment:
Some comment/docstring on why the first value in time is the correct one would be useful. Why can we ignore the others?

Reply:
I don't know why we can ignore the other value (there are 2 values in the log).

Reply:
(I just noticed that Jochens code used only the first value, and when I did the same the results overlapped)

Reply:
Then that should at the very least be documented here!